db/migrations/2019_11_19_000000_add_api_privilege.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Engelsystem\Migrations;
+
+use Engelsystem\Database\Migration\Migration;
+
+class AddApiPrivilege extends Migration
+{
+    /**
+     * Run the migration
+     */
+    public function up()
+    {
+        $connection = $this->schema->getConnection();
+
+        // Add api permissions
+        $connection->insert(
```

Comment:
`$this->schema->getConnection()->table('User')->insert([ [...]` should be used here instead of a less flexible sql query, see other migrations for reference

Reply:
I actually copied code from another migration (2018_01_01_000004_cleanup_group_privileges.php)  ;)
You still got a point though.

Reply:
Yeah, some of the old migrations need to be cleaned up too ;)